src/ui/components/search/searchcomponent.js

```diff
@@ -547,29 +552,28 @@ export default class SearchComponent extends Component {
                 this._geolocationOptions)
             );
           } else {
-            return this.search(query);
+            return this.search(query, searchOptionsOverrides);
           }
         });
     } else {
-      return this.search(query);
+      return this.search(query, searchOptionsOverrides);
     }
   }
 
   /**
    * Performs a query using the provided string input.
    * @param {string} query The string to query against.
+   * @param {Object} searchOptionsOverrides The options to pass for core search
    * @returns {Promise} A promise that will perform the query and update globalStorage accordingly.
    */
-  search (query) {
+  search (query, searchOptionsOverrides) {
```

Comment:
This is a weird function signature. I was considering making it just `searchOptions` and passing it from the initial call of `debouncedSearch` but I was worried about moving the logic of `if verticalKey searchOptions1 else searchOptions2` to every occurrence.

Reply:
Still on the fence.

Reply:
I changed it